concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -263,6 +263,11 @@ resources:
       uri: https://github.com/alphagov/paas-compose-scraper
       tag_filter: v0.1.0
 
+  - name: paas-usage-events-collector
+    type: git
+    source:
+      uri: https://github.com/alphagov/paas-usage-events-collector
```

Comment:
Are we happy not to specify a tag/hash in order to avoid deploying changes as soon as they hit `master`?

Reply:
We should pin to a specific version, yes. I think the pipelines in release-ci will automatically tag things in Git for you. I think we do this for the compose broker for example.

Reply:
You are right, I created two PRs which should be merged first to build the first release tag for the project.

Please review and merge first:
 - https://github.com/alphagov/paas-usage-events-collector/pull/3
 - https://github.com/alphagov/paas-release-ci/pull/39

Reply:
@bandesz you've got a release tag now: https://github.com/alphagov/paas-usage-events-collector/commit/5aab946182a8ec5ec4bdf8ff24f46f9428b14bea

Reply:
@samcrang version pinned in a new commit